lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
@detiber Ah, ok, I see what you're saying now.

I still don't think we want lookup to have a 'default' to set, though.

Instead, I think we want the lookup plugin to have the same param as the default filter, where we can tell it to skip values that evaluate to false.

Something like this:
```
# Before
"{{ lookup('oo_option', 'image_name   default=CentOS-7-x86_64-GenericCloud.qcow2') }}"

# My proposal
"{{ lookup('oo_option', 'image_name   skip_empty=true') | default('CentOS-7-x86_64-GenericCloud.qcow2', true) }}"
```

Thoughts?